src/gu/sceGuLight.c

```diff
@@ -10,15 +10,21 @@
 
 void sceGuLight(int light, int type, int components, const ScePspFVector3 *position)
 {
-	GuLightSettings* settings = &light_settings[light];
+	int offset = light * 3;
+	int ctype;
 
-	sendCommandf(settings->xpos,position->x);
-	sendCommandf(settings->ypos,position->y);
-	sendCommandf(settings->zpos,position->z);
+	if (components == GU_POWERED_DIFFUSE) {
+		ctype = GU_DIFFUSE_AND_SPECULAR;
```

Comment:
GU_DIFFUSE matches 2, which was the default before.
```suggestion
		ctype = GU_DIFFUSE;
```